autoscheduler/frontend/src/tsconfig.json

```diff
@@ -4,7 +4,7 @@
         "sourceMap": true,
         "noImplicitAny": true,
         "module": "commonjs",
-        "target": "es6",
+        "target": "es2018",
```

Comment:
Just to clarify, this was needed for the `new Map()` you're using for the termMap?

Reply:
It's needed for the function `Object.entries(obj)`, technically we only need to use es2017 for that function but I chose to use 2018 because that's what our `eslintrc` was already configured to